public/instructors_list.php

```diff
@@ -85,6 +110,73 @@
 
 <br><br>
 
+<h2>Courses assigned to Instructors</h2>
+<?php if ($course_assignment && count($course_assignment)) { ?>
+    <table>
+            <thead>
+                <tr>
+                    <th>Course Offering</th>
+                    <th>Instructor</th>
+                    <th>Created At</th>
+                </tr>
+            </thead>
+            <tbody>
+        <?php foreach ($course_assignment as $row) { 
+            $inst = User::find_by(array('id' => $row->user_id));
+            $off = CourseOffering::find_by(array('id' => $row->offering_id));
```

Comment:
You shouldnt be doing this! You should define an association between `CourseOfferingInstructor` and the `User` & `CourseOffering` classes

Reply:
you mean like do a join between the classes?

Reply:
er.. not exactly, I meant that you could do 'CourseOfferingInstructor' belongs to 'User', same for `CourseOffering`